src/components/badge-list/badge-list.module.css

```diff
@@ -1,9 +1,4 @@
-/**
```

Comment:
We probably don't want to delete this copyright blurb, right?

Reply:
Yep I'll have to add it back... I have to add it to all of the new files as well.

Reply:
Actually looks like a bot will come through and add this in after the fact; I'll let that manage it!